flair/datasets/sequence_labeling.py

```diff
@@ -447,6 +447,53 @@ def __init__(
             document_separator_token=None if not document_as_sequence else "-DOCSTART-",
         )
 
+class MIT_Restaurants(ColumnCorpus):
```

Comment:
Can you capitalize this to `MIT_RESTAURANTS`? We currently capitalize all prepared corpora.

Reply:
Done.
[New test file - FlairTask.py](https://megantosh.s3.eu-central-1.amazonaws.com/MITRestoCorpus/FlairTask.py)